packages/service-utils/src/core/api.ts

```diff
@@ -0,0 +1,55 @@
+import type { ServiceName } from "./services";
+
+export type CoreServiceConfig = {
+  apiUrl: string;
+  serviceScope: ServiceName;
+  serviceApiKey: string;
+  serviceAction?: string;
+};
+
+export type ApiKeyMetadata = {
+  id: string;
+  key: string;
+  creatorWalletAddress: string;
+  secretHash: string;
+  walletAddresses: string[];
+  domains: string[];
+  bundleIds: string[];
+  services: {
+    name: string;
+    targetAddresses: string[];
+    actions: string[];
+  }[];
+};
+
+export type ApiResponse = {
+  data: ApiKeyMetadata | null;
+  error: {
+    code: string;
+    statusCode: number;
+    message: string;
+  };
+};
+
+export async function fetchKeyMetadataFromApi(
+  clientId: string,
+  config: CoreServiceConfig,
+): Promise<ApiResponse> {
+  const { apiUrl, serviceScope, serviceApiKey } = config;
+  const url = new URL(`${apiUrl}/api/v1/keys`);
```

Comment:
its no longer /use?

Reply:
yeah I was just wondering why it no longer works :D

Reply:
fixing